front-end/src/app/shared/models/schc1-transaction-type.model.ts

```diff
@@ -0,0 +1,35 @@
+import { TransactionType, TransactionTemplateMapType } from './transaction-type.model';
+
+export abstract class SchC1TransactionType extends TransactionType {
+  scheduleId = 'C1';
+
+  // Mapping of schedule fields to the group input component form templates
+  templateMap: TransactionTemplateMapType = {
+    last_name: '',
+    first_name: '',
+    middle_name: '',
+    prefix: '',
+    suffix: '',
+    street_1: 'lender_street_1',
+    street_2: 'lender_street_2',
+    city: 'lender_city',
+    state: 'lender_state',
+    zip: 'lender_zip',
+    employer: '',
+    occupation: '',
+    organization_name: 'lender_organization_name',
+    committee_fec_id: '',
+    date: 'loan_incurred_date',
+    dateLabel: 'DATE',
+    memo_code: 'memo_code',
+    amount: 'loan_amount',
+    amountInputHeader: 'Loan Information',
+    aggregate: '',
+    purpose_description: '',
+    purposeDescripLabel: '',
+    text4000: 'text4000',
+    category_code: '',
+    election_code: 'election_code',
+    election_other_description: 'election_other_description',
```

Comment:
I think election_code and election_other_description can be empty ''. I don't see those fields in the schedule C1 spec

maybe the same for memo_code and text400... we can add them back if they turn up on the input form. I'd rather remove them and have to add them later as if we leave them we might forget

Reply:
I election_code, election_other_description, and memo_code to empty.

I kept text4000, as I *expect* that that's necessary.